client/lib/plans/features-list.js

```diff
@@ -218,7 +218,7 @@ export const FEATURES_LIST = {
 		getTitle: () => i18n.translate( '200 GB storage' ),
 		getDescription: () =>
 			i18n.translate(
-				"With increased storage space you'll be able to upload more images, videos, audio, and documents to your website."
+				'Upload more images, audio, and documents to your website with increased storage space.'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'With increased storage space you’ll be able to upload more images, audio, and documents to your website.' )` *ES Score: 9*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/plan-page-tooltip-copy)